.gitignore

```diff
@@ -0,0 +1,9 @@
+practice/.idea/compiler.xml
```

Comment:
All files from `.idea` will be ignored when you ignore the `.idea` directory. 
You can ignore all `.iml` file with: `*.iml`